libsolidity/ast/Types.cpp

```diff
@@ -949,7 +1042,10 @@ TypePointer RationalNumberType::binaryOperatorResult(Token::Value _operator, Typ
 			else if (other.m_value > numeric_limits<uint32_t>::max())
 				return TypePointer();
 			uint32_t exponent = other.m_value.numerator().convert_to<uint32_t>();
-			value = rational(m_value.numerator() / pow(bigint(2), exponent), 1);
+			if (exponent > msb(m_value.numerator()))
```

Comment:
`msb` is an extremely short name to be used without namespace. Is that imported somewhere?

Reply:
I don't think this will be a problem - ``msb`` is not imported in the namespace anyways, but only found via argument-dependent lookup and, if there will be a disambiguity at some point, this should raise an error and can be adjusted then.